ci/testnet-manager.sh

```diff
@@ -207,27 +207,30 @@ start() {
   testnet-beta)
     (
       set -x
-      # Force delete the network, as two different cloud providers are being used
-      # (otherwise, a subset of the nodes stay up while first cloud is deleted,
-      #  and the leader might be in that subset)
-      NO_VALIDATOR_SANITY=1 \
-      RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com ec2 us-west-1a \
-          -t "$CHANNEL_OR_TAG" -n 35 -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 -D
-      NO_VALIDATOR_SANITY=1 \
-      RUST_LOG=solana=info \
-        ci/testnet-deploy.sh beta-testnet-solana-com gce us-west1-a \
-          -t "$CHANNEL_OR_TAG" -n 65 -c 0 -x -P -D
+
+      # List of zones to deploy for each cloud provider
+      GCE_ZONES=(us-west1-b asia-east2-a europe-west4-a southamerica-east1-b us-east4-c)
+      EC2_ZONES=(sa-east-1a us-west-1a ap-northeast-2a eu-central-1a ca-central-1a)
+
+      # Build a string to pass as opts to testnet-deploy.sh: "-z zone1 -z zone2 ..."
+      for val in "${GCE_ZONES[@]}"; do
+        GCE_ZONE_ARGS="-z $val $GCE_ZONE_ARGS"
```

Comment:
This loop will reverse the order of zones. Maybe better to retain the original order. The bootstrap leader is the first node in the final config. We can control it to be in us-west1-b by maintaining the order